packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -142,6 +181,62 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
     this.ariaMessage = message;
   };
 
+  private setIsCollapsed = (isCollapsed: boolean) => {
+    this.isCollapsed = isCollapsed;
+  };
+
+  private toggleClass(element: Element, className: string, condition: boolean) {
+    if (condition) {
+      element.classList.add(className);
+    } else {
+      element.classList.remove(className);
+    }
+  }
```

Comment:
Unless I am mistaken, you can use [classList.toggle](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle) to achieve the same result.

```
element.toggle(className, condition)
```

Reply:
so much cleaner thanks!